build/images/ipfix-collector/README.md

```diff
@@ -1,14 +1,13 @@
 # images/ipfix-collector
 
-This Docker image is based on Ubuntu 18.04 which includes an IPFIX collector based on [libipfix](http://libipfix.sourceforge.net/), a C library.
-In this image, IPFIX collector listening on tcp:4739 port.
+This Docker image is based on Ubuntu 18.04 which includes an IPFIX collector based on [go-ipfix](https://github.com/vmware/go-ipfix), a Golang library.
+In this image, IPFIX collector is listening on tcp:4739 port.
 
-libipfix package is downloaded from https://svwh.dl.sourceforge.net/project/libipfix/libipfix/libipfix-impd4e_110224.tgz
+Go-ipix collector package is from `github.com/vmware/cmd/collector`.
 
 New version of the image can be built and pushed to Dockerhub using following instructions:
 
 ```bash
-cd build/images/ipfix-collector
```

Comment:
how do we specify docker file location here?

Reply:
My bad. This should not be deleted. Thanks!